src/app/frontend/logs/logstoolbar/logstoolbar_controller.js

```diff
@@ -0,0 +1,164 @@
+     */
+    this.pod = this.findPodByName_(this.pods, this.params.podId);
+
+    /** @export {!Array<!backendApi.PodContainer>} */
+    this.containers = [];
+
+    /** @export {!backendApi.PodContainer} */
+    this.container;
+
+    /**
+     * Pod creation time.
+     * @export {string}
+     */
+    this.podCreationTime;
+
+    /**
+     * Namespace.
+     * @private {string}
+     */
+    this.namespace = this.params.namespace;
+
+    /**
+     * Replica Set name.
+     * @private {string}
+     */
+    this.replicaSetName = this.params.replicaSet;
+
+    /**
+     * Flag indicates state of log area color.
+     * If false: black text is placed on white area. Otherwise colors are inverted.
+     * @export {boolean}
+     */
+    this.isTextColorInverted = false;
+
+    if (angular.isDefined(this.pod)) {
+      this.podCreationTime = new Date(Date.parse(this.pod.startTime)).toLocaleString();
+      this.containers = this.pod.podContainers || [];
+      this.container = this.findContainerByName_(this.containers, this.params.container);
+    }
+  }
+
+  /**
+   * Execute a code when a user changes the selected option of a pod element.
+   * @param {string} podId
+   * @return {string}
+   * @export
+   */
+  onPodChange(podId) {
+    // TODO(zreigz): state name and StateParam can not be import from logstoolbar_state. Issue 153.
+    return this.state_.transitionTo("logs", {
+      namespace: this.namespace,
+      replicaSet: this.replicaSetName,
+      podId: podId,
+      container: this.container.name,
+    });
+  }
+
+  /**
+   * Execute a code when a user changes the selected option of a container element.
+   * @param {string} container
+   * @return {string}
+   * @export
+   */
+  onContainerChange(container) {
+    // TODO(zreigz): state name and StateParam can not be import from logstoolbar_state. Issue 153.
+    return this.state_.transitionTo("logs", {
+      namespace: this.namespace,
+      replicaSet: this.replicaSetName,
+      podId: this.pod.name,
+      container: container,
+    });
+  }
+
+  /**
+   * Execute a code when a user changes logs panel color.
+   * @export
+   */
+  onTextColorChange() {
+    this.isTextColorInverted = !this.isTextColorInverted;
+    this.logManagementFactory.notifyTextColorChange();
+  }
+
+  /**
+   * Find object by name in array objects.
+   * Return object or undefined if can not find a object.
+   * @param {!Array<T>} array
+   * @param {!string} name
+   * @return {T}
+   * @template T
+   * @private
+   */
+  findByName_(array, name) {
+    return array.filter(function(obj) {
+      if (obj.name === name) {
+        return obj;
+      }
+    })[0];
+  }
+
+  /**
+   * Find Pod by name.
+   * Return object or undefined if can not find a object.
+   * @param {!Array<!backendApi.ReplicaSetPodWithContainers>} array
+   * @param {!string} name
+   * @return {!backendApi.ReplicaSetPodWithContainers}
+   * @private
+   */
+  findPodByName_(array, name) { return this.findByName_(array, name); }
```

Comment:
I've added something like this
```
  findPodByName_(array, name) {
    return array.prototype.find((element) => { return element.name === name; });
  }
```
but I have error because ```find``` method can not be found. According to documentation ```find```  may not be available in all JavaScript implementations yet. Should I add the snippet from documentation anywhere ?